app/assets/src/components/CliUserInstructionsModal.jsx

```diff
@@ -0,0 +1,191 @@
+import React from "react";
+import { Modal, Button, Form, TextArea } from "semantic-ui-react";
+import PropTypes from "prop-types";
+
+class CliUserInstructionsModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { open: false };
+    this.close = this.close.bind(this);
+    this.open = this.open.bind(this);
+  }
+
+  // These are needed since Modal.open = this.state.open
+  open() {
+    this.setState({ open: true });
+  }
+
+  close() {
+    this.setState({ open: false });
+  }
+
+  render() {
+    const singleUploadCmd = `idseq -e ${this.props.email} -t ${
```

Comment:
be consistent with strings definition: `` ` `` -> `"` unless you need to avoid escaping quotes.